applications/models.py

```diff
@@ -90,9 +90,17 @@ class BootcampApplicationQuerySet(models.QuerySet):
     """Custom queryset for BootcampApplication model"""
 
     def prefetch_state_data(self):
-        """Prefetches models that inform the state of bootcamp applications"""
-        return self.select_related("user__profile").prefetch_related(
-            "submissions", "orders", "bootcamp_run__application_steps__application_step"
+        """
+        Prefetches models that inform the state of bootcamp applications,
+        and filters to only include fulfilled orders.
+        """
+        return self.select_related("user__profile", "bootcamp_run").prefetch_related(
```

Comment:
`bootcamp_run` is added here because it improves performance. From [the docs](https://docs.djangoproject.com/en/3.0/ref/models/querysets/), specifically this line: "Of course, the best_pizza relationship could also be fetched using select_related to reduce the query count to 2:"